scripts/find_unique_isoform.py

```diff
@@ -8,6 +8,7 @@
 import edlib
 import numpy
 import logging
+import logging
```

Comment:
There are a lot of repeated lines. This might be due to an automatic merging. Could you resolve this?

Reply:
hmmmm...yes, probably true.

I didn't intentionally change anything in `scripts`, but did do some merging with `main` and whatnot -- i must have messed something up there.

If this pull is going to be accepted, then anything upstream of finding unique isoforms is obsolete. For downstream functions, I'll make sure that I have an up-to-date and correct version, and then integrate it into `src` and the cmd line interface before the pull is accepted/merged.